### DISCUSSION TITLE
Fedora RPM build - Yes, it works

### DISCUSSION THREAD

**Discussion by plvlnisse** [abraunegg/onedrive#1519] [Show and tell]
I built opendrive-2.4.12 using the CentOS recipie on the web site. Only one minor quirk that I had to fix manually:

```
$ diff -bup onedrive.spec.orig onedrive.spec
--- onedrive.spec.orig	2021-06-18 12:30:34.993401697 +0200
+++ onedrive.spec	2021-06-18 12:33:23.969094489 +0200
@@ -59,7 +59,7 @@ make
 %{_docdir}/%{name}
 %{_bindir}/%{name}
 %if 0%{?with_systemd}
-%{_unitdir}/%{name}.service
+%{_userunitdir}/%{name}.service
 %{_unitdir}/%{name}@.service
 %else
 %{_bindir}/onedrive_service.sh

```
This was Fedora-33 so it may show up in future CentOS/RHEL...

The original spec file was [https://raw.githubusercontent.com/abraunegg/onedrive/master/contrib/spec/onedrive.spec.in](https://raw.githubusercontent.com/abraunegg/onedrive/master/contrib/spec/onedrive.spec.in)

**Comment by abraunegg** [comment]
@plvlnisse 
Thanks for the feedback regarding the spec file.

For reference, refer to the following RedHat bugzilla id for Fedora packages: https://bugzilla.redhat.com/show_bug.cgi?id=1911017 

For Fedora RPM's of v2.4.12 also refer to: https://koji.fedoraproject.org/koji/taskinfo?taskID=68913865

To rebuild for Fedora 33, you are better off using the srpm from above and using 'mock' via `mock --rebuild` to rebuild the srpm for your Fedora 33 platform.
